test/suite/hook_tiered.py

```diff
@@ -102,7 +163,7 @@ def session_alter_replace(orig_session_alter, session_self, uri, config):
     # of a tiered table.  Note this isn't a precise match for when we did/didn't create
```

Comment:
Can we just add a comment saying the "ALter isn't implemented for tiered tables or read-only tiered tests"? I know it is obvious from the if condition but this is just my thought, feel free to ignore it.